### DISCUSSION TITLE
How to get Windows 10+ Acrylic Blur on a window?

### DISCUSSION THREAD

**Discussion by fitdev** [AvaloniaUI/Avalonia#13226] [Q&A]
I used the steps from the [docs](https://docs.avaloniaui.net/docs/next/tutorials/music-store-app/creating-a-modern-looking-window#acrylic-blur), however all I get is a window with dark gray background.

I tried running the app in debug and release modes, even published it with `<RuntimeIdentifier>win-x64</RuntimeIdentifier>` specified as was suggested somewhere. But still I cannot get the window transparency to work. Even `TransparencyLevelHint="Blur"` does not work either.

I am on Windows 11 22H2 x64.

**Comment by timunie** [comment]
show your xaml of the window pls

**Reply by fitdev** [reply]
It's just xaml from the docs:

```xaml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="AvaloniaAcrylicTest.MainWindow"
        Title="AvaloniaAcrylicTest"
        TransparencyLevelHint="AcrylicBlur"
        Background="Transparent"
        >
    <Panel>
           <ExperimentalAcrylicBorder IsHitTestVisible="False">
               <ExperimentalAcrylicBorder.Material>
                   <ExperimentalAcrylicMaterial
                       BackgroundSource="Digger"
                       TintColor="Black"
                       TintOpacity="1"
                       MaterialOpacity="0.65" />
               </ExperimentalAcrylicBorder.Material>
           </ExperimentalAcrylicBorder>
       </Panel>
</Window>
```

And the csproj:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net7.0</TargetFramework>
    <Nullable>enable</Nullable>
    <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
    <ApplicationManifest>app.manifest</ApplicationManifest>
    <AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>
    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
  </PropertyGroup>


  <ItemGroup>
    <PackageReference Include="Avalonia" Version="11.0.0" />
    <PackageReference Include="Avalonia.Desktop" Version="11.0.0" />
    <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.0" />
    <PackageReference Include="Avalonia.Fonts.Inter" Version="11.0.0" />
    <!--Condition below is needed to remove Avalonia.Diagnostics package from build output in Release configuration.-->
    <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.0.0" />
  </ItemGroup>
</Project>
```

**Reply by timunie** [reply]
please try to update ava version

**Reply by fitdev** [reply]
Sorry, I just copy-pasted for a new sample project that used older version. The same thing is happening with the latest 11.0.4 version. I just explicitly verified it by doing a Release publish for win-x64, and I get window with dark gray background.

Here's the repro: [AvaloniaAcrylicTest.repro.zip](https://github.com/AvaloniaUI/Avalonia/files/12884707/AvaloniaAcrylicTest.repro.zip)

**Reply by JaredMerritt** [reply]
This happens for me too :)

**Reply by maxkatz6** [reply]
Make sure that you have up-to-date windows. Actylic blur is not supported with any windows 10 version.

**Reply by JaredMerritt** [reply]
On Windows 11 here, have also tried blur with no luck

**Reply by fitdev** [reply]
Just in case, check that you have these effects enabled in Windows Settings (I think under System Performance or somewhere like this)
